packages/MapFactory/src/DTK_C_API_Map.hpp

```diff
@@ -242,13 +282,13 @@ DTK_Map *createMap( DTK_ExecutionSpace map_space, MPI_Comm comm,
             {
             case DTK_HOST_SPACE:
                 map = new DTK_MapImpl<Serial, HostSpace, HostSpace>(
-                    comm, source, target );
+                    comm, source, target, ptree );
```

Comment:
I think this is the right way to do it - let the map builders handle the options themselves.